fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/mixin/registry/sync/MixinRegistry.java

```diff
@@ -35,16 +33,6 @@
 	@Unique
 	private final EnumSet<RegistryAttribute> attributes = EnumSet.noneOf(RegistryAttribute.class);
 
-	@Shadow
-	public abstract Set<Identifier> getIds();
-
-	/**
-	 * This is used to denote backwards compatibility, when false default attributes will be applied.
-	 * This is set to true when using FabricRegistryBuilder
-	 */
-	@Unique
-	private boolean builtWithBuilder = false;
```

Comment:
Screw backwards compatibility at this point, I guess?

Reply:
Yes, an API is needed to add a registry now, so no point in assuming the old way is there anymore. If mods have updated in the paste few days then they should be active enought to move to the API.